lua/no-neck-pain/main.lua

```diff
@@ -59,23 +59,24 @@ local function createBuf(name, cmd, padding, moveTo)
 
     vim.api.nvim_win_set_width(0, padding)
 
-    if _G.NoNeckPain.config.buffers.showName then
+    if _G.NoNeckPain.config.showBufferNames then
         vim.api.nvim_buf_set_name(0, "no-neck-pain-" .. name)
     end
 
-    for scope, _ in pairs(_G.NoNeckPain.config.buffers.options) do
-        for key, value in pairs(_G.NoNeckPain.config.buffers.options[scope]) do
-            vim[scope][key] = value
-        end
+    -- there is probably a cleaner way to do this but it's kind of annoying
+    -- since the tables contain multiple different kinds of values
+    for opt, val in pairs(_G.NoNeckPain.config.buffers[name].bo) do
+        vim.bo[opt] = val
     end
 
-    vim.cmd(moveTo)
+    for opt, val in pairs(_G.NoNeckPain.config.buffers[name].wo) do
+        vim.wo[opt] = val
+    end
 
-    if _G.NoNeckPain.config.buffers.background.colorCode ~= nil then
-        D.print("CreateWin: setting `colorCode` for buffer " .. id)
+    vim.cmd(moveTo)
 
-        C.init(id, _G.NoNeckPain.config.buffers.background.colorCode)
-    end
+    C.init(id, _G.NoNeckPain.config.buffers.left.color)
+    C.init(id, _G.NoNeckPain.config.buffers.right.color)
```

Comment:
```suggestion
    C.init(id, _G.NoNeckPain.config.buffers[name].color)
```

The function is called for each side so we can just do it like this